src/main/java/TableReader.java

```diff
@@ -0,0 +1,26 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class TableReader {
+
+
+    protected int[][] readTableFromFile(String path, int rows, int maxCol) {
```

Comment:
Jeśli użyłaś nazwy "maxCol" to nasuwa się, żeby użyć też "maxRow" :)

Reply:
Nie ma za co przepraszać, mam nadzieję, że zdana :)